Source/FieldSolver/FiniteDifferenceSolver/EvolveM.cpp

```diff
@@ -38,6 +38,8 @@ void FiniteDifferenceSolver::EvolveM (
         amrex::Abort("Unknown algorithm");
     } */
 
+    amrex::Abort("The function EvolveM should not be used for macroscopic materials");
+
```

Comment:
Shouldn't this say "non-macroscopic materials"?

Reply:
This is EvolveM.cpp, which is meant to be used for non-macroscopic materials.